### ISSUE TITLE
[Bug][a11y] When talkback is enabled the view-new-collection snackbar is missing

### ISSUE THREAD

**Issue by codrut-topliceanu** [mozilla-mobile/fenix#16592]
## Steps to reproduce
1. Enable Talkback
2. Open a few tabs
3. Save one or more tabs to a collection

### Expected behavior
After clicking "OK" a snackbar should appear (lasting around 15 seconds) with the text "tab saved" or "collection saved" and a "View" action button.

### Actual behavior
No snackbar appears

### Device information

* Android device: 11
* Fenix version: 
<img src="https://user-images.githubusercontent.com/60002907/99245376-23c89a80-280c-11eb-8a98-214c15b092ab.png" width="300"/>

**Comment by sflorean** [comment]
Verified as fixed on Pixel 3(Android 11) on Nightly 11/25.
